client/src/document/organisms/sidebar/index.tsx

```diff
@@ -28,6 +28,16 @@ export function SidebarContainer({ doc, children }) {
     }
   }, [isSidebarOpen]);
 
+  useEffect(() => {
+    const sidebar = document.querySelector(".sidebar");
+    const currentSidebarItem = sidebar?.querySelector("em");
+    if (sidebar && currentSidebarItem) {
+      sidebar.scrollTo({
+        top: currentSidebarItem.offsetTop - window.innerHeight / 3,
+      });
+    }
+  }, [isSidebarOpen]);
```

Comment:
This isn't doing anything currently, and with my other suggested fix means that the scroll is reset every time the menu is opened and closed - which is very annoying.
```suggestion
  }, []);
```